src/components/atoms/AchievementComponent/AchievementComponent.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styles from "./AchievementComponent.module.scss";
+import Image from "next/image";
+import Button from "../button/Button";
+
+interface AchievementProps {
+  title: string;
+  description: string;
+  image: string;
+}
+
+const AchievementComponent: React.FC<AchievementProps> = ({
```

Comment:
Could we consider renaming this component to "AchievementCard" for better clarity?